crates/bevy_sprite/src/bundle.rs

```diff
@@ -13,6 +13,8 @@ use bevy_transform::components::{GlobalTransform, Transform};
 #[derive(Bundle, Clone, Default)]
 pub struct SpriteBundle {
     pub sprite: Sprite,
+    /// Defines if how the sprite behaves along its size
```

Comment:
Not proper english. Perhaps just change to

/// Controls how the image is altered when scaled.

Reply:
```suggestion
    /// Controls how the image is altered when scaled.
```